### DISCUSSION TITLE
Multiple Copies

### DISCUSSION THREAD

**Discussion by chanjs2** [lairdubois/lairdubois-opencutlist-sketchup-extension#569] [Q&A]
Hello,

I was wondering if I wanted to make multiple of the same design, what kind of option I should apply? If there is no such option, would there be another way to make this process easier?

Thank you!

**Comment by bbeaulant** [comment]
Hi @chanjs2 ,

There’s no special option. Just duplicate your design in your model and OpenCutList will use copies.

have a nice day.
